components/collective-navbar/index.js

```diff
@@ -284,11 +317,13 @@ const CollectiveNavbar = ({
   showBackButton,
   withShadow,
   useAnchorsForCategories,
+  router,
 }) => {
+  const newNavbarFeatureFlag = get(router, 'query.version') === 'v2';
```

Comment:
Maybe we should be more explicit with this flag, ie:

```suggestion
  const newNavbarFeatureFlag = get(router, 'query.navbarVersion') === '2';
```